plugins/automation/app/serializers/discourse_automation/automation_serializer.rb

```diff
@@ -84,6 +85,25 @@ def trigger
       }
     end
 
+    def include_stats?
+      scope&.dig(:stats).present?
+    end
+
+    def stats
+      automation_stats = scope&.dig(:stats, object.id) || {}
+
+      {
+        last_day: automation_stats[:last_day] || empty_stats,
+        last_week: automation_stats[:last_week] || empty_stats,
+        last_month: automation_stats[:last_month] || empty_stats,
+        last_run_at: automation_stats[:last_run_at],
+      }
+    end
+
+    def empty_stats
```

Comment:
Could perhaps just be defined as a constant.

Reply:
that is true, will move it